apache-couchdb-mixin/panels.libsonnet

```diff
@@ -0,0 +1,351 @@
+        + g.panel.timeSeries.standardOptions.withUnit('reqps'),
+
+      overviewTemporaryViewReadsPanel:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Temporary view reads',
+          targets=[signals.overview.temporaryViewReads.asTarget() { interval: '2m' }],
+        )
+        + g.panel.timeSeries.panelOptions.withDescription('The total number of temporary view reads aggregated across all nodes.')
+        + g.panel.timeSeries.standardOptions.withUnit('rps'),
+
+      overviewRequestMethodsPanel:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Request methods',
+          targets=[signals.overview.requestMethods.asTarget() { interval: '2m' }],
+        )
+        + g.panel.timeSeries.panelOptions.withDescription('The request rate split by HTTP Method aggregated across all nodes.')
+        + g.panel.timeSeries.standardOptions.withUnit('reqps')
+        + g.panel.timeSeries.options.legend.withAsTable(true)
+        + g.panel.timeSeries.options.legend.withPlacement('right'),
+
+      overviewRequestLatencyPanel:
+        g.panel.histogram.new(title='Request latency quantiles')
+        + g.panel.histogram.queryOptions.withTargets([
+          signals.overview.requestLatency.asTarget()
+          + g.query.prometheus.withInstant(true)
+          + g.query.prometheus.withFormat('timeseries'),
+        ])
+        + g.panel.histogram.panelOptions.withDescription('The request latency aggregated across all nodes.')
+        + g.panel.histogram.standardOptions.color.withMode('thresholds')
+        + g.panel.histogram.fieldConfig.defaults.custom.stacking.withMode('normal')
+        + g.panel.histogram.options.withBucketCount(60)
+        + g.panel.histogram.options.legend.withAsTable(true)
+        + g.panel.histogram.options.legend.withPlacement('right')
+        + g.panel.histogram.standardOptions.withUnit('s'),
+
+      overviewBulkRequestsPanel:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Bulk requests',
+          targets=[signals.overview.bulkRequests.asTarget() { interval: '2m' }],
+        )
+        + g.panel.timeSeries.panelOptions.withDescription('The total number of bulk requests aggregated across all nodes.')
+        + g.panel.timeSeries.standardOptions.withUnit('rps'),
+
+      overviewResponseStatusOverviewPanel:
+        g.panel.pieChart.new('Response status overview')
+        + g.panel.pieChart.queryOptions.withTargets([
+          signals.overview.responseStatus2xx.asTarget() { interval: '2m' },
+          signals.overview.responseStatus3xx.asTarget() { interval: '2m' },
+          signals.overview.responseStatus4xx.asTarget() { interval: '2m' },
+          signals.overview.responseStatus5xx.asTarget() { interval: '2m' },
+        ]) + g.panel.pieChart.panelOptions.withDescription('The responses grouped by HTTP status type (2xx, 3xx, 4xx, and 5xx) aggregated across all nodes.'),
```

Comment:
Could you mention here that by default all nodes are shown, and more usage may be found by selecting a single node in the drop down?

Reason being pie-charts can't stack statuses properly, so if everything is all green, you get as many "2xx" slices as there are nodes, which looks a bit odd.